extensions/wikia/Insights/scripts/InsightsIndexPageTracking.js

```diff
@@ -0,0 +1,26 @@
+require(['wikia.tracker'],
+	function (tracker) {
+		'use strict';
+
+		function onLinkClick() {
+			/* Track a click on an insights type link */
+			var trackingParams = {
+				trackingMethod: 'both',
+				category: 'insights-index',
+				action: tracker.ACTIONS.CLICK_LINK_TEXT,
+				label: $(this).parent().data('type')
+			};
+			tracker.track(trackingParams);
+		}
+
+		function init() {
+			/* Setup click events on all links within special insights container section */
+			var $insightsContainer = $('.insights-container-landing');
+			$insightsContainer.find('a').each(function(){
```

Comment:
Can we use ```on``` function and bind click event once? I will also use 'mousedown' event